changelogs/fragments/fix-snmp-server-idempotency.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - Fixed idempotency issue in SNMP configuration management, preventing unintended overwrites and ensuring accurate updates.
```

Comment:
This can be more elaborate with what specific issue it is trying to solve, this is a bit vague.

Reply:
Hi i updated it in new commit